exercises/concept/secrets/.docs/introduction.md

```diff
@@ -0,0 +1,24 @@
+# Lambdas
+
+Python has a [`lambda`][lambdas] expression, which is a shorter way to create functions than using `def`. It looks like this:
+
+```python

```

Comment:
Assigning a `lambda` to a variable is a [Python Anti-pattern](https://docs.quantifiedcode.com/python-anti-patterns/correctness/assigning_a_lambda_to_a_variable.html)